util/encoding.go

```diff
@@ -0,0 +1,31 @@
+package util
+
+import (
+	"bytes"
+	"encoding/binary"
+)
+
+// Helpers functions regarding the encoding of basic types ( integers ...)
```

Comment:
Yes but I don't see the utility in having helper functions like this that return fresh byte-arrays; it seems both more efficient and just as clean and simple either to (a) just store directly into a caller-supplied byte array as LittleEndian.PutUint64 does, if the caller wants to handle layout directly; or (b) use something like abstract.Read/abstract.Write instead, which handles the layout itself into a Buffer or other stream object.